admin_app/src/app/integrations/components/ChatManagerModalContents.tsx

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+// import YouTube from 'react-youtube';
+
+import DownloadIcon from "@mui/icons-material/Download";
+import { Box, Button, Typography } from "@mui/material";
+import Link from "next/link";
+
+import { Layout } from "@/components/Layout";
+import { sizes } from "@/utils";
+
+// make one of these for each chat manager and use it to populate the modal
+const ChatManagerContentExample: React.FC = () => {
+  return (
+    <Layout.FlexBox flexDirection={"column"} gap={sizes.doubleBaseGap}>
+      <Typography variant="subtitle1">Example Title</Typography>
+      <Typography
+        variant="body2"
+        sx={{
+          overflowWrap: "break-word",
+          hyphens: "auto",
+          whiteSpace: "pre-wrap",
+        }}
+      >
+        Example Text
+      </Typography>
+    </Layout.FlexBox>
+  );
+};
+
+const TypebotModalContent: React.FC = () => {
+  return (
+    <Layout.FlexBox flexDirection={"column"} gap={sizes.baseGap}>
+      <Typography variant="body1">
+        In Typebot, you can use the "HTTPS request" card to call AAQ and receive
+        its response.
+      </Typography>
+      <Typography variant="body1" mb={sizes.baseGap}>
+        To get a head start, you can download our template below and load it
+        into Typebot by going to "Create a typebot &gt; Import a file".
+      </Typography>
+
+      <Box display="flex" justifyContent="center">
+        <Button
+          variant="outlined"
+          color="primary"
+          startIcon={<DownloadIcon />}
+          href="https://github.com/IDinsight/aaq-core/blob/e75c7fec20373a8db5f5c8771b069aa7fac576a8/chat_managers/typebot/llm_response_flow.json"
```

Comment:
Thoughts on connecting to the raw one so it doesn't open github?

https://raw.githubusercontent.com/IDinsight/aaq-core/e75c7fec20373a8db5f5c8771b069aa7fac576a8/chat_managers/typebot/llm_response_flow.json

Downside is that they'll have to figure out how to download it instead of the nice button in github